.circleci/config.yml

```diff
@@ -4,31 +4,36 @@ jobs:
     docker:
       - image: linuxbrew/linuxbrew
     environment:
-      CIRCLE_REPOSITORY_URL: https://github.com/linuxbrew/homebrew-xorg
       HOMEBREW_DEVELOPER: 1
       HOMEBREW_NO_AUTO_UPDATE: 1
       HOMEBREW_VERBOSE: 1
       HOMEBREW_VERBOSE_USING_DOTS: 1
     steps:
-      - run: |
-          cd /home/linuxbrew/.linuxbrew/Homebrew
-          git fetch origin --unshallow --tags
-          git reset --hard origin/master
-      - checkout
-      - run: git remote set-url origin $CIRCLE_REPOSITORY_URL
-      - run: if [ -e .git/shallow ]; then echo git fetch --unshallow; fi
-      - run: git fetch origin
-      - run: git config --global user.name LinuxbrewTestBot
-      - run: git config --global user.email testbot@linuxbrew.sh
-      - run: mkdir -p /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
-      - run: cp -a . /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew/$CIRCLE_PROJECT_REPONAME
-      - run: |
-          mkdir /tmp/bottles
-          cd /tmp/bottles
-          umask 022
-          PATH="/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH"
-          brew install patchelf
-          brew test-bot --tap=linuxbrew/xorg --bintray-org=linuxbrew --git-name=LinuxbrewTestBot --git-email=testbot@linuxbrew.sh
+      - run:
+          name: Set up Git
+          command: |
+            git config --global user.name LinuxbrewTestBot
+            git config --global user.email testbot@linuxbrew.sh
+      - run:
+          name: Set up linuxbrew/xorg tap
+          working_directory: /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/linuxbrew
+          command: |
+            git clone https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
```

Comment:
Not sure we should keep `$CIRCLE_PROJECT_USERNAME` or replace it with a shorter and clearer `Linuxbrew`

Reply:
I'm okay with `Linuxbrew`.

Reply:
and what about `CIRCLE_PROJECT_REPONAME`? 
```
$ echo -n '$CIRCLE_PROJECT_REPONAME' | wc -c
24
$ echo -n 'homebrew-xorg' | wc -c
13
```

Reply:
Hmmm... I guess we can keep `CIRCLE_PROJECT_REPONAME` because this way file can be used for all Linuxbrew repos